Schema/KissOpenpubSkill.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$id": "https://kissdevelopment.commonground.nu/kiss_openpub_skill.schema.json",
```

Comment:
I was unsure on what this url is based, I took the kiss development gateway url.